train_seq2seq.py

```diff
@@ -0,0 +1,217 @@
+from __future__ import print_function
```

Comment:
actually, make this just be in train.py ; add parseargs for selecting model and config. Can you modify it so we can load the config class w values from a JSON file? Should be pretty easy, just use __dict__ and do a for loop over keys